docs/transformations/ExpressionUnification.md

```diff
@@ -0,0 +1,32 @@
+# Expression Unification
+
+## Overview
+
+This transformation unifies binary expressions with commutative integer operands.
+
+## Restrictions
+
+* operands are only integer
+* operators are only ["*", "+", "|", "&", "^"]
+
+## Examples
+* 
+Before:
+```python
+3 * x * 2 * 1
+```
+After:
+```python
+1 * 2 * 3 * x
+```
+
+* 
+
+Before:
+```python
+'h' + (4 + 9 * 1)
```

Comment:
What about this case:
```
b + c + a
```
where a, b and c is int?
I checked this case, it works, but, please, add information about this case in this doc and add some tests for this case. I understand that this will not work if the type is not int or cannot be obtained, but still, we should add this in the tests and in the description